cmd/subnetcmd/deploy.go

```diff
@@ -258,6 +260,25 @@ func controlKeysLoop(controlKeysPrompt string, network models.Network) ([]string
 				continue
 			}
 			controlKeys = append(controlKeys, controlKey)
+		case delCtrlKey:
+			if len(controlKeys) == 0 {
+				fmt.Println("no keys added yet")
+				continue
+			}
+			ifaceK := make([]interface{}, len(controlKeys))
+			for i, v := range controlKeys {
+				ifaceK[i] = v
+			}
+			index, err := app.Prompt.CaptureIndex("Choose key to remove:", ifaceK)
+			if err != nil {
+				return nil, false, err
+			}
+			controlKeys = append(controlKeys[:index], controlKeys[index+1:]...)
+		case preview:
+			fmt.Println("Control keys:")
+			for i, k := range controlKeys {
+				fmt.Printf("%d. %s\n", i, k)
```

Comment:
probably use ux here

Reply:
I actually disagree, ux prints both to log and stdout. Logs such as the above will make no sense in the log as there is no context, so I think it is ok to just print to stdout directly as it is.

Applies to all the other occurrences as well.